### DISCUSSION TITLE
What is libusb and what is used for?

### DISCUSSION THREAD

**Discussion by seltix5** [yungd1plomat/AdvancedSharpAdbClient#117] [Q&A] [answered]
Hello,
I noticed this reference to libusb but I do not see any use of it in the code.
What is the use of this?

https://github.com/SharpAdb/AdvancedSharpAdbClient/blob/77db1e933f98ac4562574d407eb5ee735f992211/AdvancedSharpAdbClient/Extensions/AdbServerFeatures.cs#L30

UPDATE: On my original post I referenced "libusb.dll" but it was my mistake because this file was generated from another NuGet project, not AdvancedSharpAdbCliente.

Thanks!

**Comment by wherewhere** [comment]
？？？

**Reply by seltix5** [reply]
updated the question with the code reference of libusb

**Reply by wherewhere** [reply, marked as answer]
That's const string to send with adb server. I don't know what it used for because it is there before I join the project...
